src/Hooks/MultilingualSupport.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\Hooks\MultilingualSupport.
+ */
+
+namespace Netzstrategen\CoreStandards\Hooks;
+
+class MultilingualSupport {
+
+  /**
+   * Changes the command language to return entities of matching language only.
+   *
+   * Currently supporting WPML only.
```

Comment:
Please add that either a locale or the literal string "all" can be passed.  And that the default language is applied if nothing is passed.